src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -791,6 +822,31 @@ private void UpdateVersionGlobalJson(DependencyDetail itemToUpdate, JToken token
             }
         }
 
+        private void UpdateDotNetToolsManifest(DependencyDetail itemToUpdate, JToken token)
+        {
+            string versionElementName = itemToUpdate.Name;
+
+            JObject toolsNode = (JObject) token["tools"];
+
+            foreach (JProperty property in toolsNode?.Children<JProperty>())
+            {
+                if (property.Name.Equals(versionElementName, StringComparison.OrdinalIgnoreCase))
+                {
+                    JValue versionEntry = (JValue) property.Value.SelectToken("version", false);
+
+                    if (versionEntry != null)
+                    {
+                        versionEntry.Value = itemToUpdate.Version;
+                    }
+                    else
+                    {
+                        _logger.LogError($"Entry found, but no version property to update, for dependency '{itemToUpdate.Name}'");
+                    }
+                    break;
```

Comment:
[nit] return right here instead of the break?

Reply:
Sure, the break exits the foreach so I think it's the same anyways.